pom.xml

```diff
@@ -323,6 +327,12 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-clients</artifactId>
+            <version>${kafka.version}</version>
```

Comment:
Shouldn't it be scoped to test as well?

Reply:
this one is problematic. It comes as transitive dependency from connect-api dependency and it has provided scope. When I import it with test scope it overrides the transitive dependency and compilation fails in some parts

Reply:
if it comes at transitive dependency, cant we directly use it?
I am afraid adding more dependency will make this jar bulkier. 

that being said, I like the change, so consider this as a non blocking comment.